src/server/actions/home-actions.js

```diff
@@ -0,0 +1,35 @@
+
+/**
+ * Dependencies.
+ */
+
+var mysql = require('mysql');
+var queries = require('./home-queries');
+
+/**
+ * Home page actions.
+ */
+
+module.exports = {
+  loadPage: function (actionContext, payload, done) {
+    var connection = mysql.createConnection({
+      host: 'localhost',
+      user: 'root',
+      password: '',
+      database: 'seattle_theatre'
+    });
+
+    connection.connect();
+
+    connection.query(queries.retrieveVenues, function(err, rows) {
```

Comment:
So is `connection.connect` synchronous? or does `connection.query` wait for the connection to be established?